homeassistant/components/uptimerobot/config_flow.py

```diff
@@ -34,9 +34,12 @@ async def _validate_input(
         """Validate the user input allows us to connect."""
         errors: dict[str, str] = {}
         response: UptimeRobotApiResponse | UptimeRobotApiError | None = None
-        uptime_robot_api = UptimeRobot(
-            data[CONF_API_KEY], async_get_clientsession(self.hass)
-        )
+        key: str = data[CONF_API_KEY]
+        if key.startswith("ur"):
```

Comment:
Should we guard this for monitor specific keys as well?

Reply:
Did you find a specific key structure for that scenario ?

Reply:
Yeah, they are simple in their naming, those starts with `m` 😄

Reply:
Done